llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -793,7 +761,9 @@ handleExtendedOpcodesForLineTable(DWARFDataExtractor &LineTableDataReader,
     if (LineTableDataReader.getAddressSize() != Len - 1)
       return createStringError(inconvertibleErrorCode(),
                                "Address size mismatch");
-    LineTableDataReader.getRelocatedAddress(LineTableCursor);
+    // matches the current address offset? Or are all DW_LNE_set_address
+    // operands always guaranteed to be an unresolved relocation.
```

Comment:
It's not clear to me whether this comment is a TODO for the implementation or whether this explains the code...

Reply:
It is not needed, I had added this while writing this for myself, but I have confirmed that it is the relocation addend value itself.